release-team/role-handbooks/docs/README.md

```diff
@@ -366,25 +368,11 @@ If no suitable candidates are available, you may choose to lead again or nominat
 
 The Docs Lead is responsible for updating the Kubernetes API reference, kubectl, and components documentation.
 
-#### Update the generated documentation 
-
-Update the generated documentation using a Python script ([Generating Reference Pages for Kubernetes Components and Tools](https://kubernetes.io/docs/contribute/generate-ref-docs/kubernetes-components/)). Before running the script, modify `reference.yml` to checkout the Kubernetes future release branch.
-
-![update kubernetes release branch](pics/reference-yaml.png)
-
-#### Update the api markdown
-
-Create the `static/docs/reference/generated/kubernetes-api/<MINOR_VERSION>` directory for the future release. Copy the current release directory and rename it.
-
-   ![copy-rename-dir](pics/copy-rename-dir.png)
-
-#### Update release links
-
-Add the future release link to `content/en/docs/reference/_index.md` and remove the oldest release link. The **API Reference** section should only contain five links.
+#### Generate the reference documentation
 
-   ![add-remove-link](pics/add-remove-link.png)
+Generate the reference documentation using the `website/updated-imported-docs/update-imported-docs.py` script ([Generating Reference Pages for Kubernetes Components and Tools](https://kubernetes.io/docs/contribute/generate-ref-docs/quickstart/)).
```

Comment:
@kbhawkey: Since this task will always happen before the release, does it make sense to include some context related to updating the generated docs with r.c?

Reply:
Sure. I'd like to keep or contain most of the information about the script and building the references in the `docs/contribute` page.
I can add something like:

When a Kubernetes release candidate version tag is created, you can consider building the reference documentation.
The reference documentation requires a valid release tag in order to build the API reference, such as `v1.19-rc.3`.
Create a branch for this work based off of the `dev-<release_num>` branch.
Run the script providing the release tag to build the reference documentation and commit the generated files.
You can expect to maintain this branch with periodic updates to the reference documentation using the latest `rc`
tag. Merge your branch to the `dev-<release_num>` branch just before the final release.
When the final release tag is created, you should update the reference documentation to the master branch using the final release tag.

**Note:** You should not have to build the reference documentation for every release candidate version.

Reply:
Sounds good to me